test/markdown-parser.js

```diff
@@ -229,4 +229,43 @@ code
 
     expect(tokens).to.deep.equal([]);
   });
+
+
+  it("should ignore jekyll front matter and not content in horizontal lines ", () => {
+    const tokens = markdownParser(`
+---
+author: test
+---
+
+---
+This should be spell checked
+---
+`);
+    expect(tokens).to.deep.equal([
+      { text: 'This', index: 27 },
+      { text: 'should', index: 32 },
+      { text: 'be', index: 39 },
+      { text: 'spell', index: 42 },
+      { text: 'checked', index: 48 }]);
+  });
+
+  it("should ignore top jekyll front matter and not front matter within the content ", () => {
```

Comment:
does this test guard against anything that the above test doesn't?

Reply:
Sort of, one of the issues we had (isullivan/_posts/2014-11-28-bramble-mvc) was having front matter within the content of the post. Granted this wasn't down to the parsing of front matter (more just the fact that it had the horizontal lines), but i thought it might be a useful test case anyway to ensure that it doesnt miss spell checking of front matter within content. Happy to remove this though as you are right it wont really catch anything different at the moment

Reply:
I think the above plus "doesn't ignore text between two horizontal rules in the middle of the content" has it covered.

If you want to keep it though, just move it up to be with the others and change the description to be more like the aforementioned one.